build-system/eslint-rules/forbidden-terms-config.js

```diff
@@ -36,11 +36,12 @@ module.exports = {
      */
     function* removeFromArray(fixer, node) {
       const {text} = context.getSourceCode();
-      let {start} = node;
+      let [start] = node.range;
+      const [end] = node.range;
```

Comment:
Shouldn't this be the second item?
```suggestion
      const [,end] = node.range;
```

Reply:
It looks like `start` is decremented on line 42, so I think the `start` and the `end` variables start at the same point while `start` is decremented until it finds the real starting point.

Reply:
In that case, should the code be changed as follows for better readability?

```js
      const [end] = node.range;
      let start = end;
```

Reply:
Justin's right. my bad!